docs/console/docs/06-03-backendmodule.md

```diff
@@ -3,7 +3,7 @@ title: BackendModule
 type: Custom Resource
 ---
 
-The BackendModule CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to enable UI API Layer modules.
+The `BackendModule` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to enable UI API Layer modules.
```

Comment:
"BackendModule" shouldn't be formatted as code. 

The introductory sentence of this document should look like this: 

> The `backendmodule.ui.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to enable UI API Layer modules.